### PR TITLE
Preference edits

### PR DESCRIPTION
**Pull request by LordFowl**
Makes several lorefriendly changes to chargen.
Permits numbers in chargen naming for the benefit of IPCs.
Permits chargen fluff to be viewed via employment record consoles and ID cards.

### CODE REVIEW

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -135,6 +135,10 @@
 	var/blood_type = "\[UNSET\]"
 	var/dna_hash = "\[UNSET\]"
 	var/fingerprint_hash = "\[UNSET\]"
+	var/citizenship = "\[UNSET\]"
+	var/faction = "\[UNSET\]"
```
Comment (skull132) [inline code/game/objects/items/weapons/cards_ids.dm]: Faction is ominous enough to where we won't use it. At least that's the present plan.

So, you can remove it from here and other places (but leave it in the prefs for the time being).

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -165,6 +165,26 @@ datum/preferences
 
 	ZeroSkills(1)
 
+/datum/preferences/proc/getMinAge(var/AGE_MIN)
```
Comment (skull132) [inline code/modules/client/preferences.dm]: All caps in var names are a no-go.

All caps indicate a `#define`.

**Comment by skull132** [comment]
DNM, two marks requiring resolution.

**Comment by LordFowl** [comment]
Updated.

**Pull request merged**